src/draw/vg_lite/lv_draw_vg_lite.c

```diff
@@ -152,7 +170,7 @@ static void draw_execute(lv_draw_vg_lite_unit_t * u)
             break;
     }
 
-    lv_vg_lite_flush(u);
+    return has_flushed;
```

Comment:
What about inverting the logic?
```
bool flush = false;
switch (...) {
   case ....:
       lv_draw_vg_lite_.....();
       flush = true;
   case ....:
   .....
}

if ( flush ) lv_vg_lite_flush(u);
return flush;
```

Changing a boolean value needs less CPU instructions than calling a function (and passing parameter).
(it's just a micro-optimization, but if above suggested logic is correct for every switch-case, then some bytes can be spared in firmware)